apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -67,9 +67,9 @@ loop(Socket, Transport, State) ->
                     do_send(Socket, Transport, B, M, T, C,
                             State#state.rr_min_time),
                     loop(Socket, Transport, State);
-                %% {get_dirty, B, M, T, C} ->
-                %%     do_send_opt(Socket, Transport, B, M, T, C, [no_rr]),
-                %%     loop(Socket, Transport, State);
+                {get, B, M, T, C, Opts} ->
```

Comment:
this is a good point, and would be addressed through https://github.com/dalmatinerdb/dproto/pull/8